crates/proto/src/serialize/binary/encoder.rs

```diff
@@ -132,7 +132,7 @@ impl<'a> BinEncoder<'a> {
         BinEncoder {
             offset: offset as usize,
             // TODO: add max_size to signature
-            buffer: private::MaximalBuf::new(u16::max_value(), buf),
+            buffer: private::MaximalBuf::new(u16::MAX, buf),
```

Comment:
FYI I just changed these while passing by since my IDE was showing that `max_value()` is deprecated.